uncharted-aske/HMI
# Enable drilldown back

- Enable drilldown panel for model components (nodes). NOTE: In the future, the drilldown panel should only be open from the local view.
- Simplify the metadata pane to display anything in metadata until we get more metadata examples. Currently, it displays whatever we get from Arizona CAGs and GrFNs without much styling.
- Extended node graph data structure in the epi graph adapter to ease access to deeply nested attributes. Before, they were nested in `data`.


![image](https://user-images.githubusercontent.com/10552785/101839187-33ed4480-3b42-11eb-956d-5654882b5d56.png)

To test:

- Select a specific model.

- Select a node from the graph. As a result, the drilldown panel should be open. If there is no metadata, an alert will get displayed at the moment. If there is metadata, it should be visible in the pane.
